### PR TITLE
[ISSUE #4092] Replaced isEqualto() by hasSize() method

### LINKED ISSUE #4092: [Enhancement] Use assertThat(actual).hasSize(expected) instead.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-sdks/eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java line 136

### Describe the solution you'd like

image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by sumitdethe27**
Fixes #4092 .

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

replaced isEqualto() by hasSize() method

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by mxsm — COMMENTED** [review]
@sumitdethe27  Please fix CI

File: eventmesh-sdks/eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java [hunk 1]
```diff
@@ -138,7 +138,7 @@ public EventMeshProtocolType getProtocolType() {
             }
         });
         eventMeshGrpcConsumer.subscribe(Collections.singletonList(buildMockSubscriptionItem()));
-        assertThat(eventMeshGrpcConsumer.getSubscriptionMap().size()).isEqualTo(1);
+        assertThat(eventMeshGrpcConsumer.getSubscriptionMap().size()).hasSize(1);
```
Comment (Alonexc) [inline eventmesh-sdks/eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java:141]:
```
assertThat(eventMeshGrpcConsumer.getSubscriptionMap()).hasSize(1);
```

Comment (sumitdethe27) [reply eventmesh-sdks/eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java:141]: Ohh i forgot to check 
Thankyou

**Review by Pil0tXia — APPROVED** [review]
LGTM

**Review by mxsm — APPROVED** [review]

**Review by Alonexc — APPROVED** [review]

**Pull request merged**